c10/util/intrusive_ptr.h

```diff
@@ -347,19 +378,12 @@ class intrusive_ptr final {
    * passed in *must* have been created using intrusive_ptr::release().
    */
   static intrusive_ptr reclaim(TTarget* owning_ptr) {
-    return intrusive_ptr(owning_ptr);
+    return intrusive_ptr(owning_ptr, dont_increase_refcount);
   }
 
   template <class... Args>
   static intrusive_ptr make(Args&&... args) {
-    auto result = intrusive_ptr(new TTarget(std::forward<Args>(args)...));
-    // We can't use retain_(), because we also have to increase weakcount
-    // and because we allow raising these values from 0, which retain_()
-    // has an assertion against.
-    ++result.target_->refcount_;
-    ++result.target_->weakcount_;
-
-    return result;
+    return intrusive_ptr(new TTarget(std::forward<Args>(args)...));
```

Comment:
You can probably delete `intrusive_ptr::make()` and instead call the constructor directly from `make_intrusive`.